E-R Travlendar - Review Dulu

nmuntaaza opened the pull request:
Hasil analisi kemarin lusa, belum di update lg berdasar analisi kemarin.

nmuntaaza committed:
E-R Travlendar

Hasil analisi kemarin lusa, belum di update lg berdasar analisi kemarin.

Dianinsani123 commented:
![image](https://user-images.githubusercontent.com/29325875/31302595-14a1adb2-ab2d-11e7-84de-ccf883db7c72.png)

sejauh ini db yang dibuat naufal sudah cukup baik.
*sebenarnya saya juga masih mempertimbangkan hal ini, untuk versi 1 karena tidak menggunakan system login mungkin untuk username, password, email tidak begitu dibutuhkan. tapi jangan dulu di hapus, karena dari pihak analis pun masih mempertimbangkan akan ada login atau tidak di versi 1. kemarin tidak di bahas akan ada login karena "login tidak begitu di prioritaskan (dlm pengerjaan) di awal.
1. untuk id kota - id provinsi bisa diganti menjadi kode_provinsi & kode_kota

nmuntaaza commented:
Terima kasih atas reviewnya. Akan ada update malam ini insyaallah. 

Saya berencana akan menambahkan kecamatan dan desa, sebagaimana studi kasus terbaru.

Dianinsani123 commented:
Oke, mungkin bisa mengadopsi beberapa entitas yg dibuat oleh Nurindah sebelumnya. Perhatikan juga setiap penamaan atribut, relasi maupun entitas. Ada konsisten dalam penggunaan bahasa.

MaulanaIbrahim commented:
bagaimana apakah sudah fix untuk database?

nmuntaaza commented:
@A2JTK16/analysis-team silahkan ikut mereview jika paham.

nmuntaaza committed:
E-R Travlendar Update v2

MaulanaIbrahim commented:
![image](https://user-images.githubusercontent.com/28476564/31313186-3d53f37c-ac05-11e7-8978-945844ccd73a.png)

maaf sebelumnya sedikit mereview , untuk penamaan relationship antara TransportationMode dengan estimationtime dan juga relationship antara entitas estimationtime dan distance belum diperbaiki

irakusnindi18 commented:
dalam pembuatan db sudah lebih baik

irakusnindi18 commented:
tapi kemarin diskusi perihal mock up nanti untuk memasukan alamat itu dalam satu entity , tapi mungkin untuk versi satu untuk lokasi seperti itu juga gapapa

nmuntaaza commented:
@irakusnindi18
Seharusnya mockup mengikuti database/model

nmuntaaza commented:
@MaulanaIbrahim
Selain penamaan ada apa lagi yang dapat di review. Jika bisa menargetkan menemukan 10 kesalahan seperti yang di saran kan para dosen.

nmuntaaza commented:
Oh ya kalo bisa ketika me review mention saya @irakusnindi18 @MaulanaIbrahim @Dianinsani123

nmuntaaza commented:
@Dianinsani123 oh ya kira-kira apa yang bisa di adopsi dari entitas yang indah buat? apa seperti nama tempat (rumah, kantor, dll)?

Dianinsani123 commented:
![image](https://user-images.githubusercontent.com/29325875/31314507-b7244b08-ac2c-11e7-93be-7c936db66773.png)
1. kenapa jadi nambah kecamatan sama desa lagi ? kan di studi kasus sudah jelas tidak ada. saran saya provinsi kota dan lokasi saja sudah mencangkup semuanya. karena didalam nm_lokasi sudah ditulis secara manual (jalan, dan kecamatan) oleh user sendiri "untuk versi satu".  @NaufalMWaliy

nmuntaaza commented:
![capture](https://user-images.githubusercontent.com/31490940/31314714-1ba1760a-ac32-11e7-973c-86ffb6330285.PNG)

@Dianinsani123 bukannya studi kasus terbaru itu sampai kampung? kalo sebagian data ada di data base dan sebagian lain di input oleh user berarti nanti akan ada dropdown dan juga text field? lalu bagaimana membandingkan jaraknya apabila terdapat alamat yang di input manual oleh user? coba kritisi lagi pendapat saya

Dianinsani123 commented:
itu sama dengan studi kasus yang lama, bahkan awal. coba saja di samakan lagi. saya punya file nya dari studi kasus awal sampai kemarin malam (sebenarnya yang di share kemarin malam itu tidak ada perubahan).

yang ada di dalam dropdown kan hanya data master saja. data mater kan sudah ada provinsi dan kota, untuk data jalan tidak akan menjadi data master
@NaufalMWaliy

nmuntaaza commented:
@Dianinsani123 lalu kalau memang begitu kira-kira bagaimana cara membuat distance matrixnya?

Dianinsani123 commented:
iya sebentar, untuk perhitungan jarak yang sifatnya masih didalam kota yang sama namun jalan nya berbeda masih saya diskusikan dengan pihak analis @NaufalMWaliy

Dianinsani123 commented:
distance matrix itu hanya untuk mengetahui jarak antar kota saja. 
jadi untuk jarak yang sifatnya masih didalam kota yang sama, itu user menginputkan secara manual jaraknya contoh : 8 km, nanti system yang akan dengan sendirinya langsung memberikan rekomendasi kendaraan @NaufalMWaliy @NurindahYuniarti2A

nmuntaaza commented:
@Dianinsani123 Saya analisis dulu.
@NurindahYuniarti2A Tolong bantu menganalisis.

Dianinsani123 commented:
oke mangga naufal, karena itu sudah langsung dari system yang dirancang oleh backend juga, konsep dari backend nya sudah begitu, kalo mau di analisis, lagi mangga. saya tunggu malam ini juga setelah magrib sebelum jam 7 malam. agar progress mockup dan backend pun bisa langsung menyesuaikan. @NaufalMWaliy @NurindahYuniarti2A

nmuntaaza commented:
@Dianinsani123 Masalahnya apa itu bisa di implementasikan dalam database? Saya tidak bisa janji jam 19.00 selesai.

Dianinsani123 commented:
bisa, saya juga sudah mempertimbangkan dengan backend dan analis. 
Rangkuman review  : 
1.  penamaan relationship antar entitas
2. tidak menggunakan entitas kecamatan dan desa - > jadi untuk keterangan jalan, nomer rumah, kecamatan dan nama desa di inputkan secara manual oleh user dan disimpan di dalam atribut nm_lokasi.
3. jika kasusnya lokasi ada di dalam kota yang sama, maka untuk jarak dari tempat satu ke tempat lain di input kan secara manual oleh user. contoh : kampus polban (tempat awal) ke Gramedia Toko Buku jln. Merdeka Bandung (tempat kedua) itu berjarak 25 KM (misalnya) maka user akan menginputkan 25 KM setelah itu system akan menampilkan rekomendasi kendaraan yang baik digunakan dari polban menuju gramedia.  
4. Distance Matrix itu untuk menghitung jarak yang ada pada data master saja, seperti antar kota.

saya beri contoh 
![contooh](https://user-images.githubusercontent.com/29325875/31316130-4076bc46-ac51-11e7-9461-93e27f64accf.png)
jadi jika memilih lokasi awal dan akhir tidak ada di data master, maka dia akan menginputkan lokasi baru tapi tidak langsung menjadi data master, hanya menginput data saja. dan saat mengisi jarak user juga akan menginputkan perhitungan jarak yang ia ketahui saat menempuh perjalanan dari lokasi awal ke akhir.
berbeda dengan saat user memilih lokasi awal dan akhir sesuai data master (Kota-Kota) maka dalam text box akan secara otomatis terisi jarak sesuai dengan yang ada pada distance matrix.

bisanya jam berapa kalo jam 19.00 tidak bisa ?

nmuntaaza committed:
Kembali ke versi sebelumnya

Maaf untuk yang TransportationMode dan Jarak belum solve, butuh diskusi.

MufidJamaluddin commented:
@NaufalMWaliy @Dianinsani123 @NurindahYuniarti2A 

Database Naufal udh bagus, 
Alangkah lbh bagus ArrivalTime dan Depaturetime jangan disimpan di Entity Tempat, supaya lebih representatif disimpan di Entity Baru.

MufidJamaluddin commented:
Apakah Depaturetime dan Arrivaltime itu Lokasi?

NurindahYuniarti2A commented:
pada saat user akan menginputkan depaature time dan arrival time, itu menginputnya dibagian mana ? apakah saat membuat event atau menentukan lokasi ?

nmuntaaza committed:
Update 2 ER

nmuntaaza committed:
Update entity Agenda 

Menghapus atribute tanggal dan mengganti tipe data waktu_mulai_event dan waktu_berakhir_event menjadi Date & Time.

nmuntaaza committed:
Ganti attribute arrval_time jadi Mandatory

Dianinsani123 commented:
departure time juga harusnya bersifat mandatory fal, push lagi @NaufalMWaliy

nmuntaaza committed:
Merubah Relationship Pada Entity

Entity Perjalanan, Entity Jarak

MufidJamaluddin commented:
Kalau depature tiime tanggal 15/10/2017 pukul 23.30 malam dan acaranya tgl 16/10/2017 pukul 01.00 pagi bagaimana?

nmuntaaza committed:
Database Travlendar SQL

nmuntaaza committed:
Update bahasa untuk model cdm travlendar

nmuntaaza committed:
Update bahasa TravlendarDB SQL

Merubah bahasa TravlendarDB menjadi bahasa inggris

Dianinsani123 commented:
![image](https://user-images.githubusercontent.com/29325875/31616263-6d9c78a8-b2b6-11e7-954e-3e21f3a5b260.png) @NaufalMWaliy 

1. ini masih, 30 provinsi hehe. kurang 4 provinsi lagi, maluku + maluku utara + papua barat + papua :) 

![image](https://user-images.githubusercontent.com/29325875/31617023-7609efb4-b2b8-11e7-93bc-39580283e20f.png)
2. transport itu data master, jadi sekalian isi datanya yaa @NurindahYuniarti2A 

nanti untuk pembaharuaanya di push di branch ini saja. terimakasih 👍

nmuntaaza committed:
CDM dan MySQL Query

Sudah diisi semua data master yang dibutuhkan

nmuntaaza committed:
.rar akan diganti dengan folder

nmuntaaza committed:
Membuat Folder di Github

nmuntaaza committed:
Create Temp.txt

nmuntaaza committed:
Add CDM & PDM Travlendar

Mengubah relasi, penamaan dari beberapa entity

nmuntaaza committed:
Create Temp.txt

nmuntaaza committed:
Add MySQL Query Sesuai CDM

nmuntaaza committed:
Delete Temp.txt

nmuntaaza committed:
Delete Temp.txt

nmuntaaza commented:
@Dianinsani123 Sudah diselesaikan semua data master yang telah di perlukan.

[TravlendarDB.zip](https://github.com/A2JTK16/Database/files/1388293/TravlendarDB.zip)

nmuntaaza committed:
Merubah relasi antara event dengan lokasi | Update

nmuntaaza committed:
Delete Temp.txt

nmuntaaza committed:
MySQL Query yang sesuai dengan CDM | Update

nmuntaaza committed:
Merge pull request #11 from A2JTK16/master

Harap di Review Lalu Merge
